login/service_whitebox_test.go

```diff
@@ -79,6 +84,20 @@ func TestValidOAuthAccessToken(t *testing.T) {
 	assert.Equal(t, identity.Username, claims.Username)
 }
 
+// Can't use test.GenerateToken() because it would introduce a cycle dependency between test and login packages
+func generateToken(ident account.Identity) (string, error) {
```

Comment:
@alexeykazakov Yes you can simplify it to centralize the code generation. I'll make a PR to your branch right now